packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -84,6 +92,16 @@ export const createDefaultChatHandlers = memoizeOne(
         }
         await chatThreadClient.sendMessage(sendMessageRequest, options as SendMessageOptions);
       },
+      /* @conditional-compile-remove(rich-text-editor-image-upload) */
+      onUploadImage: async function (image: ArrayBuffer | Blob, imageFilename: string): Promise<UploadChatImageResult> {
```

Comment:
Do we want to allow both Blob and ArrayBuffer here?

Reply:
> Do we want to allow both Blob and ArrayBuffer here?

That's the signature from the chat SDK. Maybe it's good to be consistent and leave the flexibility?

Reply:
The passthrough function is available on the underlying stateful client if needed and the return type is UI library specific
I would think at the UI level we only need the blob support?